spec/system/distribution_system_spec.rb

```diff
@@ -20,7 +20,10 @@
         fill_in "Comment", with: "Take my wipes... please"
 
         expect do
-          click_button "Save", match: :first
+          accept_alert do
+            click_button "Save", match: :first
+          end
+          sleep 1.second
```

Comment:
`sleep` usually is a bad idea in acceptance specs, not sure if the project is using this elsewhere but I would use something like:

```ruby
expect(page.find("blah")).to have_content "Saved with success."
```

Or similar. This will make capybara wait the message to appear before giving up.

Reply:
Huum, really! Thanks!